misk/src/main/resources/misk/admin/html/home.html

```diff
@@ -0,0 +1,10 @@
+<!DOCTYPE html>
```

Comment:
We almost definitely need to allow this to be customized by the application.

We also likely need some sort of templating support for HTML content. That could be a separate PR with some sort of resource resolver that takes the name of an admin resource and a data class/bean/map containing the template parameters, and writes to an output stream (i.e. a library class not an action)

Reply:
Yep, I was looking at templating engines. I think with this I mostly want to get the serving of static files set up, and then focus on building out the admin side of things.